examples/react-demo/src/AppWrappedInIdentityKit.tsx

```diff
@@ -13,19 +13,25 @@ const mockedSignerProviderUrl = import.meta.env.VITE_MOCKED_SIGNER_PROVIDER_URL
 const nfidSignerProviderUrl = import.meta.env.VITE_MOCKED_NFID_SIGNER_PROVIDER_URL
 
 const targetCanister = import.meta.env.VITE_TARGET_CANISTER
+// eslint-disable-next-line turbo/no-undeclared-env-vars
+const isDev = import.meta.env.DEV
```

Comment:
good catch, agree, i wanted to use out of box vite env for this purpose, but it will also be prod when built in dev, and will be dev as supposed after dev build, so i agree